### DISCUSSION TITLE
Wikijs container keeps restarting in Docker

### DISCUSSION THREAD

**Discussion by mocnidule** [requarks/wiki#4868] [Help / Questions]
I am trying to follow [this article](https://docs.requarks.io/en/install/digitalocean) and set up https for wikijs. 

When I create container with command bellow:

`docker create --name=wiki -e LETSENCRYPT_DOMAIN=wiki.mywebsite.com -e 
LETSENCRYPT_EMAIL=myemail@gmail.com 
-e SSL_ACTIVE=1 -e DB_TYPE=postgres -e DB_HOST=db -e DB_PORT=5432 -e 
DB_PASS_FILE=/etc/wiki/.db-secret -v /etc/wiki/.db-secret:/etc/wiki/.db-secret:ro -e 
DB_USER=wiki -e DB_NAME=wiki -e UPGRADE_COMPANION=1 --restart=unless-stopped -h wiki 
--network=wikinet -p 80:3000 -p 443:3443 requarks/wiki:2`

Run container and do docker logs wiki, I get following output:

`Loading configuration from /wiki/config.yml... OK
DB_PASS_FILE is defined. Will use secret from file.
>>> Failed to read Docker Secret File using path defined in DB_PASS_FILE env 
variable!
EISDIR: illegal operation on a directory, read`

And the container keeps restarting.

I even removed --restart=unless-stopped when creating, but that did not help.

**Comment by NGPixel** [comment]
Does the file `/etc/wiki/.db-secret` exists? Is it accessible by docker?

The process crash because it can't read that file.
